web/src/main/react-dashboard/src/components/TestRunView.js

```diff
@@ -49,563 +42,570 @@ import {FAIL, SUCCESS, ERROR, PENDING, RUNNING, HTTP_UNAUTHORIZED, LOGIN_URI, TE
+                                                                            thickness={8}/>
+                                                                    </div>
+                                                            }
+                                                        })()}
+                                                    </td>
+                                                    <td style={{
+                                                        fontSize: "15px",
+                                                        wordWrap: "break-word",
+                                                        whiteSpace: "wrap",
+                                                        textDecoration: "none"
+                                                    }}><a href={"#" + data.scenarioDescription}>
+                                                        {data.scenarioDescription}
+                                                    </a>
+                                                    </td>
+                                                    <td
+                                                        style={{
+                                                            width: "15%",
+                                                            textAlign: "center",
+                                                            color: "#189800",
+                                                            fontSize: "20px",
+                                                            wordWrap: "break-word",
+                                                            whiteSpace: "wrap"
+                                                        }}>{data.totalSuccess}</td>
+                                                    <td
+                                                        style={{
+                                                            width: "15%",
+                                                            textAlign: "center",
+                                                            color: "#c12f29",
+                                                            fontSize: "20px",
+                                                            wordWrap: "break-word",
+                                                            whiteSpace: "wrap"
+                                                        }}>{data.totalFail}</td>
+                                                    <td
+                                                        style={{
+                                                            width: "15%",
+                                                            textAlign: "center",
+                                                            fontSize: "20px",
+                                                            wordWrap: "break-word",
+                                                            whiteSpace: "wrap"
```

Comment:
hard-coded css within js files.

Reply:
Shall we raise a separate ticket and handle this?